src/types/Deliverables.ts

```diff
@@ -4,7 +4,13 @@ export type DeliverableTypeType = 'Document';
 
 export type DeliverableCategoryType = 'All' | 'Legal' | 'Financial' | 'GIS' | 'Forestry' | 'Stakeholder Engagement';
 // export type DeliverableCategories = ['All', 'Legal', 'Financial', 'GIS', 'Forestry', 'Stakeholder Engagement'];
-export type DeliverableStatusType = 'Not Submitted' | 'In Review' | 'Rejected' | 'Approved';
+export type DeliverableStatusType =
+  | 'Not Submitted'
+  | 'In Review'
+  | 'Rejected'
+  | 'Approved'
+  | 'Needs Translation'
+  | 'Not Needed';
```

Comment:
Added these two as they were in the designs/prd. We can update it later with BE types.